---
bool success
string message
geometry_msgs/Point calculated_gripper_position